lib/internal/util.js

```diff
@@ -322,10 +322,11 @@ function join(output, separator) {
   return str;
 }
 
-// About 1.5x faster than the two-arg version of Array#splice().
+// Depending on the size of the array, this is anywhere between 1.5-10x
+// faster than the two-arg version of Array#splice()
```

Comment:
Please add the V8 version this is tested with. And there is a new faster version that is currently deactivated. See https://bugs.chromium.org/p/v8/issues/detail?id=7221

Reply:
Addressed